src/main/java/uk/gov/hmcts/cmc/claimstore/appinsights/AppInsightsTool.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.cmc.claimstore.appinsights;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.ClaimStoreApplication;
+
+import static uk.gov.hmcts.cmc.claimstore.appinsights.AppInsights.REFERENCE_NUMBER;
+
+@Component
+public class AppInsightsTool implements CommandLineRunner {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AppInsightsTool.class);
+
+    @Autowired
+    private AppInsights appInsights;
+
+    @Override
+    public void run(String... args) throws Exception {
+        LOGGER.info("Tracking events");
+        trackEvent(AppInsightsEvent.PAID_IN_FULL, 10); //1103);
```

Comment:
This does not look correct. 
Each time, application will restart it will add additional events specified in this param which is incorrect. Also how did this number is calculated as `10` or `1103`?